components/content/synthwave/components/postprocessing.vue

```diff
@@ -0,0 +1,41 @@
+<script lang="ts" setup>
+import { WebGLRenderer, Camera, Scene } from 'three'
+import { BloomEffect, ScanlineEffect, EffectComposer, RenderPass, EffectPass } from 'postprocessing'
+import { useWindowSize } from '@vueuse/core'
+let effectComposer: EffectComposer
+
+function setup(renderer: WebGLRenderer, scene: Scene, camera: Camera) {
```

Comment:
Is `@tresjs/post-processing` not covering the use-case? What would be missing?

Reply:
There's no reason except that I hadn't used `post-processing`. :) I'll switch things over.